docs/celestial.rst

```diff
@@ -0,0 +1,132 @@
+**********************************
+Regular celestial images and cubes
+**********************************
+
+One of the most common types of data to reproject are celestial images or
+n-dimensional data (such as spectral cubes) where two of the axes are
+celestial. There are several existing algorithms that can be used to
+reproject such data:
+
+* **Interpolation** (such as nearest-neighbor, bilinear, biquadratic
+  interpolation and so on). This is the fastest algorithm and is suited to
+  common use cases, but it is important to note that it is not flux
+  conserving, and will not return optimal results if the input and output
+  pixel sizes are very different.
+
+* **Drizzling**, which consists of determining the exact overlap fraction of
+  pixels, and optionally allows pixels to be rescaled before reprojection.
+  More details can be found `here <here>`__. This method is more accurate
+  than interpolation but is only suitable for images where the field of view
+  is small so that pixels are well approximated by rectangles in world
+  coordinates. This is slower but more accurate than interpolation for small
+  fields of view.
+
+* Computing the **exact overlap** of pixels on the sky by treating them as
+  **four-sided spherical polygons** on the sky and computing spherical polygon
+  intersection. This is essentially an exact form of drizzling, and should be
+  appropriate for any field of view. However, this comes at a significant
+  performance cost. This is the algorithm used by the `Montage
+  <http://montage.ipac.caltech.edu/index.html>`_ package, and we have
+  implemented it here using the same core algorithm.
+
+Currently, this package implements interpolation and spherical polygon
+intersection.
+
+The reprojection/resampling is always done assuming that the image is in
+**surface brightness units**. For example, if you have an image with a constant
+value of 1, reprojecting the image to an image with twice as high resolution
+will result in an image where all pixels are all 1.
```

Comment:
What if someone wants to measure flux (not surface brightness) on the reprojected image?
They need a utility function or at least advice to do this, no?